src/shared/locales/enUs/translations/patient/index.ts

```diff
@@ -190,6 +190,14 @@ export default {
       },
     },
     visit: 'Visit',
+    history: {
+      label: 'History',
+      eventDate: 'Event Date',
+      recordType: 'Record Type',
+      information: 'Information',
+      noHistoryTitle: 'No History',
+      noHistoryMessage: 'There is no histories for this patient',
```

Comment:
Nitpick, but we should change `noHistoryMessage` to either "There is no history for this patient" or "There are no histories for this patient" (preferably the former).